pythoncalculator/subtract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y
```

Comment:
Please fix this typo, maybe it should be 
```suggestion
    return x - y
```
or:
```suggestion
    return y - x
```